### PR TITLE
Remove dependency on CoreDNS to redirect mitmproxy dns traffic

### PR DESCRIPTION
**Pull request by 121GWJolt**
MITM proxy is capable of working with DNS requests, so I removed CoreDNS to simplify the runtime requirements. I also removed the CoreDNS instructions section from documentation, and changed affected documentation to reflect this. Also removed it from the start.bat and added an extra search method to find mitmproxy for android devices.

Since start-ios.bat doesn't have any CoreDNS in there anyway, it has been currently left as-is.

### CODE REVIEW

**Comment by 121GWJolt** [comment]
I added a hotfix here for NGINX proxy to allow underscores in headers through, as apparently the pinball game has some of these (notably, during onboarding)

**Comment by 121GWJolt** [comment]
Also... I made the TTL for the dns entries smol on purpose cuz I figured it's just local to MITMproxy and it'll probably be fine, but I am well-aware it should be greater than that by a LARGE amount normally LOL

Just makes it easier in case things need to change.

File: scripts/mitm-redirect-traffic.py
Comment (Duosion) [inline scripts/mitm-redirect-traffic.py:1]: Going to make some changes to this file for rooted android.

**Review by Duosion — APPROVED** [review]
Awesome, thanks!
I am going to merge this into a new branch in order to make some changes before merging it into development.

**Pull request merged**